dts/arm/nxp/nxp_rw6xx_common.dtsi

```diff
@@ -361,6 +395,12 @@
 		};
 	};
 
+	hci: hci_ble {
+		compatible = "nxp,hci-ble";
```

Comment:
Just a reminder for the future: This node and associated binding will need to be converted to a real HCI driver instance node along with #72323

Reply:
Thanks for the information @jhedberg. When is this expected on the mainline ? I'll work on the adoption of this rework when it's ready.

Reply:
> Thanks for the information @jhedberg. When is this expected on the mainline ? I'll work on the adoption of this rework when it's ready.

Not sure. Remains to be seen if it'll make it before the next release or if it'll only go in after it. Anyway, it shouldn't prevent new HCI drivers from being merged meanwhile (it just means that the list of drivers needing converting keeps growing)